x/stake/keeper/distribute.go

```diff
@@ -0,0 +1,127 @@
+package keeper
+
+import (
+	"fmt"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/stake/types"
+)
+
+func (k Keeper) DistributeBeaconChainInBreathBlock(ctx sdk.Context, bscFee int64) {
```

Comment:
Too much duplicated code for this function, any possibility we can reuse the `DistributeSideChainInBreathBlock`?

Reply:
Tried to do so in the beginning, but it will make this function ugly. There are many condition branches of different BEP and chainId, makes the function hard to understand.
So I use a separate function to make it more clear.